binaries/speed_benchmark_torch.cc

```diff
@@ -21,6 +21,8 @@
 #include "caffe2/core/timer.h"
 #include "caffe2/utils/string_utils.h"
 #include <torch/csrc/autograd/grad_mode.h>
+#include <torch/csrc/jit/mobile/module.h>
```

Comment:
The include file is probably need be guarded by the macro `BUILD_LITE_INTERPRETER`

Reply:
I checked 
`ANDROID_ABI=arm64-v8a ./scripts/build_android.sh -DBUILD_BINARY=ON` works ok with those includes